protoc-gen-pbandk/lib/src/commonMain/kotlin/pbandk/gen/Namer.kt

```diff
@@ -27,14 +27,14 @@ public interface Namer {
         private val disallowedFieldNamePrefixes = setOf(
             "decodeFrom", "encodeTo"
         )
-        private val disallowedValueTypeNames = disallowedTypeNames + setOf(
-            "UNRECOGNIZED"
-        )
         private val kotlinKeywords = setOf(
             "as", "break", "class", "continue", "do", "else", "false", "for", "fun", "if", "in",
             "interface", "is", "null", "object", "package", "return", "super", "this", "throw",
             "true", "try", "typealias", "typeof", "val", "var", "when", "while"
         )
+        private val disallowedValueTypeNames = disallowedTypeNames + kotlinKeywords + setOf(
```

Comment:
Since I wasn't sure if this is needed, I removed it from this PR. If you have an example that requires this, let's add it as a unit test and reintroduce this change.